libevm/JitVM.cpp

```diff
@@ -149,12 +149,15 @@ int64_t call(
 		u256 gas = _msg->gas;
 		// ExtVM::create takes the sender address from .myAddress.
 		assert(fromEvmC(_msg->sender) == env.myAddress);
-		auto addr = env.create(value, gas, input, {});
+		h160 addr;
+		owning_bytes_ref revertOutput;
+		std::tie(addr, revertOutput) = env.create(value, gas, input, {});
```

Comment:
At the moment it is. EVM-C does not support "revert output" yet so there is not option to pass this back to EVMJIT. But EVM-C v4 changes are in progress.